README.md

```diff
@@ -62,6 +65,15 @@ into the `/stable-diffusion/src/gfpgan/experiments/pretrained_models` directory.
 Download [RealESRGAN_x4plus.pth](https://github.com/xinntao/Real-ESRGAN/releases/download/v0.1.0/RealESRGAN_x4plus.pth) and [RealESRGAN_x4plus_anime_6B.pth](https://github.com/xinntao/Real-ESRGAN/releases/download/v0.2.2.4/RealESRGAN_x4plus_anime_6B.pth).
 Put them into the `stable-diffusion/src/realesrgan/experiments/pretrained_models` directory. 
 
+### LDSR
+Quadruple your resolution using Latent Diffusion, to install:
+  - Git clone https://github.com/devilismyfriend/latent-diffusion into your stable-diffusion-main/src/ folder
+  - Rename latent-diffusion-main folder to latent-diffusion
+  - If on windows: run download_models.bat to download the required model files
+  - Otherwise to manually install the model download [project.yaml](https://heibox.uni-heidelberg.de/f/31a76b13ea27482981b4/?dl=1) and [last.cpkt](https://heibox.uni-heidelberg.de/f/578df07c8fc04ffbadf3/?dl=1) and rename last.ckpt to model.ckpt

```

Comment:
Where are these links hosted? Could they fail under load? 
Asking cause, I want to add an auto downloader of moderls to the colab version of this

Reply:
why no changes are done for the past 24 hours? is @hlky okey?

Reply:
The man needs sleep lol

Reply:
Looks like the university of Heidelberg lol, idk, it's the links I got off the latent diffusion repo which are probably provided by compVis